scripts/release_scripts/deploy.py

```diff
@@ -479,6 +479,9 @@ def execute_deployment():
     if '.' in current_release_version:
         raise Exception('Current release version has \'.\' character.')
 
+    assert len(current_release_version) <= 25, (
+        'Length of version should be less than equal to 25 characters.')
```

Comment:
The length of the "version" arg should be less than or equal to 25 characters.

Reply:
Done.